src/convertToObject.js

```diff
@@ -1,18 +1,28 @@
 'use strict';
 
 /**
- * Implement convertToObject function:
- *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
- * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
- *
  * @param {string} sourceString
  *
  * @return {object}
  */
+
 function convertToObject(sourceString) {
-  // write your code here
+  const styleLines = sourceString.trim().split('\n');
```

Comment:
I think when you'll use trim() there will be no new lines (\n), maybe try to split it with `;`, then split will return you style lines
```suggestion
  const styleLines = sourceString.trim().split(';');
```